pkg/addons/seaweedfs/install.go

```diff
@@ -4,66 +4,68 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"time"
 
 	"github.com/pkg/errors"
 	"github.com/replicatedhq/embedded-cluster/pkg/helm"
 	"github.com/replicatedhq/embedded-cluster/pkg/helpers"
 	"github.com/replicatedhq/embedded-cluster/pkg/spinner"
 	corev1 "k8s.io/api/core/v1"
-	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/wait"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
 func (s *SeaweedFS) Install(ctx context.Context, kcli client.Client, hcli helm.Client, overrides []string, writer *spinner.MessageWriter) error {
-	if err := s.createPreRequisites(ctx, kcli); err != nil {
-		return errors.Wrap(err, "create prerequisites")
-	}
+	return s.Upgrade(ctx, kcli, hcli, overrides)
+}
 
-	values, err := s.GenerateHelmValues(ctx, kcli, overrides)
-	if err != nil {
-		return errors.Wrap(err, "generate helm values")
+func (s *SeaweedFS) Uninstall(ctx context.Context, kcli client.Client) error {
```

Comment:
i assume helm uninstall isn't sufficient?

Reply:
only real benefit is if there are non-namespaced resources as helm would delete those. other note is just that we're using helm to install / upgrade. not strong objection though to keep it this way.

Reply:
What if i tried to helm uninstall and on failure delete the namespace?